docs/1.4/extensions/heading-permalinks.md

```diff
@@ -63,6 +63,8 @@ This controls the HTML you want to appear inside of the generated `<a>` tag.  Us
 
 By default, we provide an embedded [Octicon link SVG](https://octicons.github.com/icon/link/), but you can replace this with any custom HTML you wish.
 
+The inner contents value is candidate for HTML filtering, so it may become empty if you filter out `<svg>` tags in HTML.
```

Comment:
Any output of this library may be subject to filtering by external HTML filtering libraries, depending on which library you're using and how you have it configured.  While this particular item may have triggered filtering in your case, I don't think this "issue" is necessarily tied to this particular feature.

If we're going to add a warning/tip like this, I think it would be more appropriate to put a note towards the bottom of [the Security page](https://commonmark.thephpleague.com/1.4/security/) recommending that anyone who chooses to use additional HTML filtering should ensure their filtering is properly configured based on the Markdown features they are using.

Reply:
Absolutely. I just wanted to write something down so this doesn't get lost. I was really confused for some moments that why example copied from documentation just did not work. In hope that someone else would be not spending time on same problem.

I don't get offended at all if you finish this up yourself :) (Allow edits from maintainers is enabled)

Reply:
Sure thing! I've force-pushed a change to put some similar documentation on the Security page instead.  I appreciate you bringing this up :)